shell/shared/grain.js

```diff
@@ -148,8 +139,6 @@ Meteor.methods({
       var grain = Grains.findOne({_id: grainId, userId: this.userId});
       if (grain) {
         Grains.remove(grainId);
-        ApiTokens.remove({grainId : grainId, $or: [{owner: {$exists: false}},
```

Comment:
My thought was that we could give a better error message if we keep them around. But I suppose that the only way that'd be useful would be to distinguish "typo in token" from "resource no longer exists", and I guess that "typo in token" is probably not going to be a common problem.